Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoLauncher.kt

```diff
@@ -32,6 +44,10 @@ class VenmoLauncher internal constructor(
         activity: ComponentActivity,
         paymentAuthRequest: VenmoPaymentAuthRequest.ReadyToLaunch
     ): VenmoPendingRequest {
+        analyticsClient.sendEvent(
+            eventName = VenmoAnalytics.APP_SWITCH_STARTED,
+            analyticsEventParams = analyticsEventParams
+        )
```

Comment:
Are we treating `venmo:tokenize:app-switch:started` as the success event for when merchants call `launch()`?

I originally thought this event was tracking when merchants invoked `launch()` and attempted to app/CCT switch rather than if `VenmoPendingRequest.Started` was returned. I can update this if my understanding is incorrect!

Reply:
Ahh hmm I guess it could be interpreted either way. I agree it is good to have an event triggered at the start of launch, but with only that we still have a gap if there is an start failure. Although the start failures should be something caught in development (invalid manifest) so maybe less necessary for analytics. We can leave it as is and add more detailed failure events if needed in the future.

Reply:
```kotlin
const val APP_SWITCH_SUCCEEDED = "venmo:tokenize:app-switch:succeeded"
const val APP_SWITCH_FAILED = "venmo:tokenize:app-switch:failed"
```

We actually have these constants already defined. Currently `APP_SWITCH_SUCCEEDED` is sent in the incorrect spot - when tokenize is called. And `APP_SWITCH_FAILED` is unused.

I also noticed that `const val TOKENIZE_STARTED = "venmo:tokenize:started"` is incorrectly sent when the merchant calls `createPaymentAuthRequest`. I'll make the fix for this.

What are your thoughts on sending `APP_SWITCH_SUCCEEDED` and `APP_SWITCH_FAILED` based on the returned `VenmoPendingRequest`?

Reply:
We use `TOKENIZE_STARTED` as the "start" event for all flows (aligns with iOS and v4), it doesn't make as much sense for v5, but I am hesitant to change it because conversion funnels might be based on that event. If we do rename it, we should do it across all payment methods and define and new conversion event in our list of FPTI events. 

I think we want to reserve `APP_SWITCH_SUCCEEDED` and `APP_SWITCH_FAILED` for the return from app switch (unless we have other events for those).

Reply:
Ah, I do see that `TOKENIZE_STARTED` is sent at the top of all the flows. I won't make any changes to this event.

| Event                               | Description                                                    |
|-------------------------------------|----------------------------------------------------------------|
| venmo:tokenize:app-switch:succeeded | Fired when we are successfully able to switch to the Venmo app |
| venmo:tokenize:app-switch:failed    | Fired when an error occurs when switching to the Venmo app     |

This is what's called out in the FPTI events doc for the app-switch succeeded and failed events. It sounds like these should be sent when the Venmo app or CCT is initially launched.

The new `const val HANDLE_RETURN_STARTED = "venmo:tokenize:handle-return:started"` event is sent when the merchant calls `VenmoLauncher.handleReturnToApp()`. We could add a handle-return succeeded and failed based on the returned `VenmoPaymentAuthResult`. Do you think it'd be worth adding these?

Reply:
I think that sounds good!

Reply:
Added the additional events here: f27b8ee75455e2de34ca9eb703b55183187da9bf